src/clusterfuzz/_internal/platforms/android/constants.py

```diff
@@ -50,3 +50,20 @@
 }
 
 RELEASE_CONFIGURATION = 'next'
+
+DEPRECATED_DEVICE_LIST = [
```

Comment:
Unfortunately these codenames on the left are internal, so they cannot be committed into open-source ClusterFuzz. We may have to find another way to determine if a testcase is considered deprecated due to the device.

Reply:
Mark, all these codenames are public at https://developers.google.com/android/drivers.

Reply:
Thanks Harish you are correct. This comment can be ignored.

Reply:
Closing this comment as per above